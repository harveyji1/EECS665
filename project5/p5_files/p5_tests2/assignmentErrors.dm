a : int;
b : bool;

fn : () void {}

func : () int {}
main : () int {
	a = true;
	a = false;
	b = 1;
	a = func();
	b = func();
	a = fn;
	b = fn;
	a = func;
	b = func;
	
}