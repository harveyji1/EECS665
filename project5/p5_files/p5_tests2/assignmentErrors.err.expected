FATAL [8,2]-[8,10]: Invalid assignment operation
FATAL [9,2]-[9,11]: Invalid assignment operation
FATAL [10,2]-[10,7]: Invalid assignment operation
FATAL [12,2]-[12,12]: Invalid assignment operation
FATAL [13,6]-[13,8]: Invalid assignment operand
FATAL [14,6]-[14,8]: Invalid assignment operand
FATAL [15,6]-[15,10]: Invalid assignment operand
FATAL [16,6]-[16,10]: Invalid assignment operand
Type Analysis Failed
